### DISCUSSION TITLE
Disabling Resource Provider stopped working in v1.19.0

### DISCUSSION THREAD

**Discussion by nkim-dev** [open-telemetry/opentelemetry-java-instrumentation#6955] [Q&A]
It looks like [disabling resource provider](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6574) via `otel.java.disabled.resource-providers` stopped working in v1.19.0
I verified it works in v1.18.x and appears to be related to the migration from sdk-extensions (#6574)

Is there a different way to disable resource providers?

**Comment by trask** [comment]
hi @nkim-dev! can you include the full JVM arg you are using to disable specific resource providers? I'm wondering if maybe the issue is due to package renames when the resource providers were moved to this repo

**Reply by nkim-dev** [reply]
Hi @trask,
Sure. Here's the JVM arg I used -
```
-Dotel.java.disabled.resource.providers=io.opentelemetry.sdk.extension.resources.ProcessResourceProvider,io.opentelemetry.sdk.extension.resources.ContainerResourceProvider,io.opentelemetry.sdk.extension.resources.OsResourceProvider,io.opentelemetry.sdk.extension.resources.ProcessRuntimeResourceProvider
```

I also tried with `io.opentelemetry.instrumentation.resources.ProcessResourceProvider,io.opentelemetry.instrumentation.resources.ContainerResourceProvider,io.opentelemetry.instrumentation.resources.OsResourceProvider,io.opentelemetry.instrumentation.resources.ProcessRuntimeResourceProvider` and yet it still does not work.

Also, tried with `-Dotel.java.disabled.resource-providers` instead of `-Dotel.java.disabled.resource.providers` and same result.
